### ISSUE TITLE
Slow convergence behavior: MACE with qGGMRF prior

### ISSUE THREAD

**Issue by dyang37** [cabouman/mbircone#123]
When testing MACE with qGGMRF denoiser, we observed that while the converged result seems correct (in the sense that it is close to MBIR recon result), the convergence is slow.
See the slides below for the experiment details (this is included in Lilly weekly update slides on 02/23/2023).
[mace_qGGMRF_slow_convergence.pptx](https://github.com/cabouman/mbircone/files/10826151/mace_qGGMRF_slow_convergence.pptx)

Per Dr. Buzzard's request, I'm recording this issue here, so that we do not lose track of it.

**Comment by cabouman** [comment]
Yes, this qGGMRF prior should be extremely useful for optimization and debugging.
